runtime/src/main/java/com/flipkart/flux/initializer/FluxInitializer.java

```diff
@@ -55,56 +62,58 @@
     );
 
 
-	/** The Polyguice DI container */
+    /**
+     * The Polyguice DI container
+     */
     private Polyguice fluxRuntimeContainer;
-    
+
     /**
      * Constructor for this class
      */
     public FluxInitializer() {
-    	try {
-    		this.hostName = InetAddress.getLocalHost().getHostName();
-    	} catch (UnknownHostException e) {
-    		//ignore the exception, not critical information
-    	}        
+        try {
+            this.hostName = InetAddress.getLocalHost().getHostName();
+        } catch (UnknownHostException e) {
+            //ignore the exception, not critical information
+        }
         this.fluxRuntimeContainer = new Polyguice();
     }
 
     /**
-     * Startup entry method 
+     * Startup entry method
      */
     public static void main(String[] args) throws Exception {
         String command = "start";
-        if (args != null && args.length> 0) {
+        if (args != null && args.length > 0) {
             command = args[0];
         }
         final FluxInitializer fluxInitializer = new FluxInitializer();
         switch (command) {
-            case "start" :
+            case "start":
                 fluxInitializer.start();
                 break;
-            case "migrate" :
+            case "migrate":
                 if (args.length < 2) {
                     throw new RuntimeException("<migrate> must be followed with db name");
                 }
-                if (!(args[1].equals("flux") || args[1].equals("flux_redriver"))) {
-                    throw new RuntimeException("<migrate> works only for 'flux' or 'flux_redriver'");
+                if (!(args[1].equals("flux") || args[1].equals("flux_scheduler"))) {
+                    throw new RuntimeException("<migrate> works only for 'flux_sharding' or 'flux_redriver'");
```

Comment:
flux_sharding? it should be flux right

minor: change flux_redriver to flux_scheduler, seems this got missed in earlier